src/coreimage.cs

```diff
@@ -4812,11 +4940,14 @@ interface CITriangleKaleidoscope : ICITriangleKaleidoscopeProtocol {
 		[CoreImageFilterProperty ("inputSize")]
 		float Size { get; set; }
 
+#if !XAMCORE_4_0
+		[Obsolete ("Use 'InputPoint0' instead.")]
```

Comment:
Shouldn't this be pointing to `InputPoint`? Or `CGPoint InputPoint { get; set; }` below should be `InputPoint0`?